c_src/emcl_nif.c

```diff
@@ -709,185 +787,198 @@ ERL_NIF_TERM enif_mcl_bn_g1_mul_vec(ErlNifEnv *env, int argc, ERL_NIF_TERM const
   return enif_return_g1(1, env, &res);
 }
 
-static
-ERL_NIF_TERM enif_mcl_bn_g1_hash_and_map_to(ErlNifEnv *env, int argc, ERL_NIF_TERM const argv[]) {
+static ERL_NIF_TERM
+enif_mcl_bn_g1_hash_and_map_to(ErlNifEnv* env, int argc, ERL_NIF_TERM const argv[])
+{
   ErlNifBinary in;
   mclBnG1 res;
 
   CHECK_BINARY_ARG(in);
 
-  if(mclBnG1_hashAndMapTo(&res, in.data, in.size)) {
+  if (mclBnG1_hashAndMapTo(&res, in.data, in.size)) {
     return error_tuple(env, "hash_and_map_to_failed");
   }
 
   return enif_return_g1(1, env, &res);
 }
 
-static
-ERL_NIF_TERM enif_mcl_bn_g1_is_equal(ErlNifEnv *env, int argc, ERL_NIF_TERM const argv[]) {
+static ERL_NIF_TERM
+enif_mcl_bn_g1_is_equal(ErlNifEnv* env, int argc, ERL_NIF_TERM const argv[])
+{
   MCL_BN_IS_EQUAL(mclBnG1, get_g1, mclBnG1_isEqual)
 }
 
-static
-ERL_NIF_TERM enif_mcl_bn_g1_is_zero(ErlNifEnv *env, int argc, ERL_NIF_TERM const argv[]) {
+static ERL_NIF_TERM
+enif_mcl_bn_g1_is_zero(ErlNifEnv* env, int argc, ERL_NIF_TERM const argv[])
+{
   MCL_BN_IS_X(mclBnG1, get_g1, mclBnG1_isZero)
 }
 
-static
-ERL_NIF_TERM enif_mcl_bn_g1_is_valid(ErlNifEnv *env, int argc, ERL_NIF_TERM const argv[]) {
+static ERL_NIF_TERM
+enif_mcl_bn_g1_is_valid(ErlNifEnv* env, int argc, ERL_NIF_TERM const argv[])
+{
   MCL_BN_IS_X(mclBnG1, get_g1, mclBnG1_isValid)
 }
 
-static
-ERL_NIF_TERM enif_mcl_bn_g1_is_valid_order(ErlNifEnv *env, int argc, ERL_NIF_TERM const argv[]) {
+static ERL_NIF_TERM
+enif_mcl_bn_g1_is_valid_order(ErlNifEnv* env, int argc, ERL_NIF_TERM const argv[])
+{
   MCL_BN_IS_X(mclBnG1, get_g1, mclBnG1_isValidOrder)
 }
 
-static
-ERL_NIF_TERM enif_mcl_bn_g1_lagrange_interpolation(ErlNifEnv *env, int argc, ERL_NIF_TERM const argv[]) {
+static ERL_NIF_TERM
+enif_mcl_bn_g1_lagrange_interpolation(ErlNifEnv* env, int argc, ERL_NIF_TERM const argv[])
+{
   // We have ensured on the Erlang side that these are non-empty lists of
   // equal length.
-  if(argc != 2 || !enif_is_list(env, argv[0]) || !enif_is_list(env, argv[1])){
+  if (argc != 2 || !enif_is_list(env, argv[0]) || !enif_is_list(env, argv[1])) {
     return enif_make_badarg(env);
   }
 
   unsigned int k;
   unsigned int i = 0;
-  if(!enif_get_list_length(env, argv[0], &k)) {
+  if (!enif_get_list_length(env, argv[0], &k)) {
     return enif_make_badarg(env);
   }
 
-  mclBnFr *xs;
+  mclBnFr* xs;
```

Comment:
No, here goes the limit also according to Ulf. This is plain wrong.

Reply:
Fixed.